### PR TITLE
SL-18977 Borders of inventory view in inventory floater seem to be cut a bit

### PR DESCRIPTION
**Pull request by LLGuru**
image
image

### CODE REVIEW

**Review by akleshchev — APPROVED** [review]

**Review by marchcat-pe — APPROVED** [review]

File: indra/newview/skins/default/xui/en/sidepanel_inventory.xml [hunk 1]
```diff
@@ -1,137 +1,113 @@
...
+     orientation="vertical">
+      <layout_panel
+       name="main_inventory_layout_panel"
+       layout="topleft"
+       follows="bottom|left|right"
+       height="300"
+       auto_resize="true"
+       user_resize="true"
+       min_dim="150">
+		<panel
```
Comment (marchcat-pe) [inline indra/newview/skins/default/xui/en/sidepanel_inventory.xml:33]: Tabs? Spaces?

File: indra/newview/skins/default/xui/en/panel_main_inventory.xml [hunk 2]
```diff
@@ -27,314 +26,298 @@
...
-      image_selected="Single_Folder_Up"
-      image_pressed="Single_Folder_Up"
-      image_unselected="Single_Folder_Up"
-      scale_image="false"
-      layout="topleft"
-      left_pad="1"
-      name="up_btn"
-      tool_tip="Go up one level"
-      width="20" />
+      <button
```
Comment (marchcat-pe) [inline indra/newview/skins/default/xui/en/panel_main_inventory.xml:59]: Formatting change?
2 space indent looks appropriate here.

**Review by marchcat-pe — APPROVED** [review]

**Comment by LLGuru** [comment]
Before the change:
image

After the change:
image

The layout is fixed in multiple levels

**Comment by LLGuru** [comment]
The scrollbars look good now:
image

File: indra/newview/skins/default/xui/en/sidepanel_inventory.xml (hunk 1 above)
Comment (LLGuru) [reply indra/newview/skins/default/xui/en/sidepanel_inventory.xml]: Spaces (fixed, thanks)

**Pull request merged**